src/Evolution/Systems/GeneralizedHarmonic/GhDampingFunctions/GaussianPlusConstant.hpp

```diff
@@ -0,0 +1,123 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines GhDampingFunctions::GaussianPlusConstant.
+
+#pragma once
+
+#include <array>
+#include <pup.h>
+
+#include "Evolution/Systems/GeneralizedHarmonic/GhDampingFunctions/GhDampingFunction.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+/// \endcond
+
+namespace GhDampingFunctions {
+template <size_t VolumeDim, typename Fr>
+class GaussianPlusConstant;
+
+/*!
+ * \ingroup GhDampingFunctionsGroup
```

Comment:
If we change the namespacing to `GeneralizedHarmonic::ConstraintDamping` then we won't need any doxygen group. By documenting the `ConstraintDamping` namespace we can find the members in the documentation by going `References->Evolution Systems->Generalized Harmonic->Constraint Damping`, which I find better because it helps keep the length of the `References` section down. Does that sound okay?